manifests/repo.pp

```diff
@@ -0,0 +1,26 @@
+class squid::repo
+{
+  if $::squid::manage_repo {
+    case $::osfamily {
+      'RedHat': {
+        case $::operatingsystemmajrelease {
+          '6','7': {
+            yumrepo { 'squid':
+              name     => 'Squid',
+              descr    => 'Squid repo for CentOS Linux - $basearch',
+              baseurl  => 'http://www1.ngtech.co.il/repo/centos/$releasever/$basearch/',
```

Comment:
@alexjfisher Well it is expected that for a storage upgrade that directory will "vanish" since the mount is not there..
Now back online and with upgraded upload speed(doubles the one before).
@*: if something changes in my services for any reason feel free to contact me via email/mobile/other.